toolkit/tools/internal/rpm/rpm.go

```diff
@@ -337,6 +337,50 @@ func QueryPackage(packageFile, queryFormat string, defines map[string]string, ex
 	return executeRpmCommand(rpmProgram, args...)
 }
 
+// QueryPackageContents queries an RPM for its file contents. The results are split into several categories:
+// - allFilesAndDirectories: all files and directories in the package
+// - files: all files in the package (ie allFilesAndDirectories minus directories)
+// - directories: all directories in the package (ie allFilesAndDirectories minus files, symlinks etc.)
+// - documentFiles: all files marked as documentation (%doc)
+// - licenseFiles: all files marked as license (%license)
+func QueryPackageContents(packageFile string, defines map[string]string,
```

Comment:
I think the name could be clearer. A package has many contents (metadata, scripts, files, signatures, etc.). If we're only doing files, why not `QueryPackageFiles` or `ListPackageFiles`?

Reply:
`QueryPackageFiles()`